release-notes.md

```diff
@@ -24,6 +38,17 @@ The following updates were made in this release:
 
 * `requestEventIdentifier` is now appended to all non-track events so that Lifecycle (or other extension events that are sent to Analytics) can be viewed with rich detail in Project Griffon.
 
+## January 24, 2020
+
+The following updates were made in this release:
+
+**Android Griffon 1.1.1**
+
+* Griffon SDK now reports event's source, type, sequence number and timestamp for every event.
```

Comment:
- Please reword as follows:
"* The Griffon SDK now reports an event's source, type, sequence number, and timestamp for every event."